app/views/includes/back-to-choices.nunjucks

```diff
@@ -1,7 +1,7 @@
 <div class="notification-banner notification-banner--hiviz js-feedback-banner">
   <p class="notification-banner--inner">
     {% if gp %}
-      This is a new page. <a class="choices-profile-link" href="{{gp.choicesProfileLink}}">See the current version of this page</a> on NHS Choices</a>.
+      This is a new page. <a class="choices-profile-link" href="{{gp.choicesProfileLink}}">See the current version</a> on NHS Choices</a>. GP staff: <a class="choices-profile-link" href="{{gp.choicesEditLink}}">sign in</a> to change information about your surgery.
```

Comment:
Has the class been added just for styling reasons? Does it link back to any analytics? I'm wondering if clicks from that link might be tracked and if they are how are they tracked? Using the class wouldn't be great given the point of a class over an id but would be good to know for sure. Also, are the number of clicks on the login link going to want to be tracked?

Reply:
It is just a styling thing.

Double checked with Shaun and they can get the click through analytics if we need them without an id or class.